pkg/services/ngalert/remote/remote_secondary_forked_alertmanager.go

```diff
@@ -76,10 +76,20 @@ func (fam *RemoteSecondaryForkedAlertmanager) TestTemplate(ctx context.Context,
 	return fam.internal.TestTemplate(ctx, c)
 }
 
-func (fam *RemoteSecondaryForkedAlertmanager) CleanUp() {}
+func (fam *RemoteSecondaryForkedAlertmanager) CleanUp() {
+	// No cleanup to do in the remote Alertmanager.
+	fam.internal.CleanUp()
+}
 
-func (fam *RemoteSecondaryForkedAlertmanager) StopAndWait() {}
+func (fam *RemoteSecondaryForkedAlertmanager) StopAndWait() {
+	fam.internal.StopAndWait()
+	fam.remote.StopAndWait()
+}
 
 func (fam *RemoteSecondaryForkedAlertmanager) Ready() bool {
-	return false
+	// Both Alertmanagers must be ready.
+	if ready := fam.remote.Ready(); !ready {
```

Comment:
You can make this simpler with:
```suggestion
	return fam.remote.Ready() && fam.internal.ready()
```